substra/sdk/client.py

```diff
@@ -949,3 +951,111 @@ def organization_info(self) -> models.OrganizationInfo:
     def cancel_compute_plan(self, key: str) -> None:
         """Cancel execution of compute plan. Nothing is returned by this method"""
         self._backend.cancel_compute_plan(key)
+
+    @logit
+    def wait_compute_plan(
+        self, key: str, *, timeout: Optional[float] = None, polling_period: float = 2.0, raises: bool = True
+    ) -> models.ComputePlan:
+        """Block the execution until the compute plan finishes.
```

Comment:
```suggestion
        """Wait for the execution of the given compute plan to finish.
```
`Block` might be a bit frightening